### DISCUSSION TITLE
specfem monthly zoom hour

### DISCUSSION THREAD

**Discussion by carltape** [SPECFEM/specfem3d#1553] [General]
The specfem team is announcing a (hopefully) monthly zoom hour for developers and users. The focus will be on coordinating development. Users are welcome to join, listen, and raise issues. If you might be interested in participating, please fill out the short google survey [here](https://docs.google.com/forms/d/e/1FAIpQLSfUDKOsyj08WybuZkLldArwycB-d9-dry_yAcNkNVsN7-5xtQ/viewform?usp=sf_link)
We will collect the email addresses and send zoom login details.

The first discussion is planned for Wed Nov 30th at 0800 US Pacific coast time, 1100 US Eastern time, 1700 central Europe.

**Comment by carltape** [comment]
Reminder: if interested, please sign up for the zoom hour discussion following the link at the start of this discussion. This first discussion will be informal: partly social, mostly code/org/community developments.

**Comment by danielpeter** [comment]
To put some items on our discussion list for this zoom meeting, this here would be something I'd like to go through quickly:
- upcoming releases: shall we clear out the git history before the releases or after?
- external source time functions: can we unify the 3 implementations in the 2D, 3D_Cartesian and 3D_Globe versions? 
- meshing: how could we mesh urban areas, i.e., buildings, with some simple approaches?

**Comment by carltape** [comment]
Hi all, it was great to see everyone on the Nov 30th zoom. It's time to schedule the next one. We're aiming for the first Wed of every month at the following times:
Alaska 0800
US Eastern 1200
Europe 1800
KAUST 2000
For January, though, we'll take the 2nd Wednesday (January 11). The one after that will be Feb 1st. If you have not previously signed up, please do so at the link at the top of this thread; I'll email the zoom details. If you previously signed up, there's no need to do so again. Thanks for your participation.

**Comment by carltape** [comment]
I made a recurring event in google calendar. Hopefully this will work:
<a target="_blank" href="https://calendar.google.com/calendar/event?action=TEMPLATE&amp;tmeid=MnR1Yjl0NGE1b2hzNDd0ZWk5N3J0Mzdhc3ZfMjAyMzAxMDRUMTcwMDAwWiBjdGFwZUBhbGFza2EuZWR1&amp;tmsrc=ctape%40alaska.edu&amp;scp=ALL"><img border="0" src="https://www.google.com/calendar/images/ext/gc_button1_en.gif"></a>
(Can someone let me know whether you see the Jan 11 and Feb 1 events?)

Or copy the link:
https://calendar.google.com/calendar/event?action=TEMPLATE&tmeid=MnR1Yjl0NGE1b2hzNDd0ZWk5N3J0Mzdhc3ZfMjAyMzAxMDRUMTcwMDAwWiBjdGFwZUBhbGFza2EuZWR1&tmsrc=ctape%40alaska.edu&scp=ALL

**Comment by brioglade** [comment]
https://paste.pics/4ca11886254b9a483fbe786e02a70eca

**Comment by carltape** [comment]
Okay, let me try again. This time I created an event on the calendar of specfem@gmail.com. You can add this calendar to yours with this link: https://calendar.google.com/calendar/u/4?cid=c3BlY2ZlbTNkQGdtYWlsLmNvbQ
You can toggle it on and off and set your own reminder notifications, as with any calendar.
A public URL to the calendar is this one:
https://calendar.google.com/calendar/embed?src=specfem3d%40gmail.com&ctz=America%2FAnchorage
The first events will be Jan 11 and Feb 1. If this does NOT work, please chime in here.

**Comment by carltape** [comment]
The next specfem zoom hour is next Wed (Jan 11th) at
0800 Alaska time
1200 US Eastern
1800 central Europe time
The zoom link will be sent to anyone who has signed up via the link at the top of the thread here. If you signed up once before, you don't need to do so again.

**Comment by danielpeter** [comment]
my agenda items: Jan-11-2023
* happy new year!
* open floor :)
* SPECFEM3D: FWI inversion example with (in-house) xinverse_problem_for_model tool
* SPECFEM3D_GLOBE: blender visualization for moviedata*** files
* containers

**Comment by bch0w** [comment]
Reminder of our upcoming SPECFEM Zoom this Wednesday Feb. 1 at

0800 Alaska Time (AKST)
1200 US Eastern Time (ET)
1800 Central Europe Time (CET)

*The zoom link will be sent to anyone who has signed up via the link at the top of the thread here. If you signed up once before, you don't need to do so again.*

--------------
Agenda:

- @Rohit-Kakodkar: Specfem Kokkos
- @bch0w: Recent adjTomo updates
- ...
- Open floor

**Reply by Rohit-Kakodkar** [reply]
To put something on the agenda, I'd like to give a brief update about specfem kokkos and timeline for the first version release.

**Comment by carltape** [comment]
The next specfem zoom hour is this Wed (March 1st) at
0800 Alaska time
1200 US Eastern
1800 central Europe time
The zoom link will be sent to anyone who has signed up via the link at the top of the thread here. If you signed up once before, you don't need to do so again. If you want to add the specfem monthly meetings to your own calendar, please see details above.

**Reply by Rohit-Kakodkar** [reply]
I'd like to brief update on Specfem2d kokkos github link - https://github.com/PrincetonUniversity/specfem2d_kokkos/tree/devel

**Comment by carltape** [comment]
The next specfem zoom hour is this Wed (April 5th) at
0800 Alaska time
1200 US Eastern
1800 central Europe time
The zoom link will be sent to anyone who has signed up via the link at the top of the thread here. If you signed up once before, you don't need to do so again. If you want to add the specfem monthly meetings to your own calendar, please see details above.

One topic I'd like to discuss pertains to using IRIS EMC netcdf models in specfem3d_globe, which has been discussed recently:
https://github.com/SPECFEM/specfem3d_globe/issues/811
Bring your topics, bring your updates, bring yourself!

**Reply by bch0w** [reply]
Some topics I'd like to discuss:

- [External velocity models in SPECFEM3D_GLOBE](https://github.com/SPECFEM/specfem3d_globe/issues/811) w/ @lsawade
- Par_file `model`='gll' behavior in SPECFEM3D_GLOBE

**Reply by danielpeter** [reply]
some more topics:
* wavefield injection (plane wave,..) in SPECFEM2D & 3D 
* ultrasonic modeling (converting DICOM tomo slices to mesh/velocity model)

**Reply by Rohit-Kakodkar** [reply]
Adding SPECFEM2D kokkos first public release and updates on performance to the list.

**Comment by homnath** [comment]
Meshing urban areas.
________________________________

**Comment by carltape** [comment]
The next specfem zoom hour is this Wed (May 3rd) at
0800 Alaska time
1200 US Eastern
1800 central Europe time
The zoom link will be sent to anyone who has signed up via the link at the top of the thread here. If you signed up once before, you don't need to do so again. If you want to add the specfem monthly meetings to your own calendar, please see details above. Come join if you have questions or suggestions about any of the specfem packages. Agenda TBD.

**Comment by carltape** [comment]
Greetings! The next specfem zoom hour is this Wed (June 7th) at
0800 Alaska time
1200 US Eastern
1800 central Europe time
The zoom link will be sent to anyone who has signed up via the link at the top of the thread here. If you signed up once before, you don't need to do so again. If you want to add the specfem monthly meetings to your own calendar, please see details above. Come join if you have questions or suggestions about any of the specfem packages. Agenda TBD -- feel free to add items or make requests here.

**Comment by carltape** [comment]
Greetings! The next specfem zoom hour is this Wed (July 5th) at
0800 Alaska time
1200 US Eastern
1800 central Europe time
The zoom link will be sent to anyone who has signed up via the link at the top of the thread here. If you signed up once before, you don't need to do so again. If you want to add the specfem monthly meetings to your own calendar, please see details above. Come join if you have questions or suggestions about any of the specfem packages. Agenda TBD -- feel free to add items or make requests here.

**Comment by carltape** [comment]
Greetings! The next specfem zoom hour is next Wed (August 2nd) at
0800 Alaska time
1200 US Eastern
1800 central Europe time
The zoom link will be sent to anyone who has signed up via the link at the top of the thread here. If you signed up once before, you don't need to do so again. If you want to add the specfem monthly meetings to your own calendar, please see details above. Come join if you have questions or suggestions about any of the specfem packages. Agenda TBD -- feel free to add items or make requests here.

**Comment by carltape** [comment]
Greetings! The next specfem zoom hour is next Wed (Sept 6th) at
0800 Alaska time
1200 US Eastern
1800 central Europe time
The zoom link will be sent to anyone who has signed up via the link at the top of the thread here. If you signed up once before, you don't need to do so again. If you want to add the specfem monthly meetings to your own calendar, please see details above. Come join if you have questions or suggestions about any of the specfem packages. Agenda TBD -- feel free to add items or make requests here.

**Comment by carltape** [comment]
Greetings! The next specfem zoom hour is next Wed (Oct 4th) at
0800 Alaska time
1200 US Eastern
1800 central Europe time
The zoom link will be sent to anyone who has signed up via the link at the top of the thread here. If you signed up once before, you don't need to do so again. If you want to add the specfem monthly meetings to your own calendar, please see details above. Come join if you have questions or suggestions about any of the specfem packages. Agenda TBD -- feel free to add items or make requests here.

**Comment by carltape** [comment]
Greetings! The next specfem zoom hour is next Wed (Nov 1st) at
0800 Alaska time
1200 US Eastern
1800 central Europe time
The zoom link will be sent to anyone who has signed up via the link at the top of the thread here. If you signed up once before, you don't need to do so again. If you want to add the specfem monthly meetings to your own calendar, please see details above. Come join if you have questions or suggestions about any of the specfem packages. Agenda TBD -- feel free to add items or make requests here.

**Comment by ljhwang** [comment]
Hi 

I am looking for figures and photos (e.g. fieldwork, workshops) from the past year for a slide loop at our AGU Business Meeting on Wednesday December 13 5:30-7:30p https://geodynamics.org/events/details/328

Please send these to me by Friday December 8 (but earlier is better)!

Thanks.

**Comment by carltape** [comment]
Greetings! The next specfem zoom hour is next Wed (Dec 6th) at
0800 Alaska time
1200 US Eastern
1800 central Europe time
The zoom link will be sent to anyone who has signed up via the link at the top of the thread here. If you signed up once before, you don't need to do so again. If you want to add the specfem monthly meetings to your own calendar, please see details above. Come join if you have questions or suggestions about any of the specfem packages. Agenda TBD -- feel free to add items or make requests here.

**Comment by carltape** [comment]
Greetings! The next specfem zoom hour is next Wed (Jan 10th) at
0800 Alaska time
1200 US Eastern
1800 central Europe time
The zoom link will be sent to anyone who has signed up via the link at the top of the thread here. If you signed up once before, you don't need to do so again. If you want to add the specfem monthly meetings to your own calendar, please see details above. Come join if you have questions or suggestions about any of the specfem packages. Agenda TBD -- feel free to add items or make requests here.

Jan 10th is also the abstract deadline for [SSA 2024 in Anchorage](https://github.com/orgs/SPECFEM/discussions/1663), a meeting that will feature multiple specfem-relevant sessions and also have [a specfem workshop](https://github.com/orgs/SPECFEM/discussions/1664) on the afternoon of Tuesday April 30th.

**Reply by Rohit-Kakodkar** [reply]
Hi All,

We just mad public the first major release for SPECFEM++, which is an effort towards unifying the suite of SPECFEM packages while providing performance portability using Kokkos. At the meeting tomorrow I plan to present a few slides and demo of the package and respond to any questions you have. Please try out the code. 

Repo link: https://github.com/PrincetonUniversity/SPECFEMPP
Documentation Link: http://specfem2d-kokkos.rtfd.io/

Hope to see everyone tomorrow.

**Reply by Rohit-Kakodkar** [reply]
Slides from today: 
[SPECFEM++_v_0_1_0.pptx](https://github.com/SPECFEM/specfem3d/files/13891385/SPECFEM%2B%2B_v_0_1_0.pptx)

If you have any questions or would like a featured added to SPECFEM++ please feel free to reach out to me at rohit_kakodkar@princeton.edu or post a issue on the SPECFEMPP github page. 

Thanks,
Rohit

**Comment by homnath** [comment]
Hi Rohit,

Sorry I had to miss today's meeting due to the conflict in schedule. That looks awesome! Really excited.

Best,
Hom Nath


---------------------------------------
Hom Nath Gharti, PhD
Assistant Professor | Digital Earth Scientist
https://www.digitalearthscience.com/
Department of Geological Sciences and Geological Engineering
Miller Hall 314, 36 Union St
Queen’s University
Kingston, ON K7L 3N6, Canada


Queen’s University is situated on traditional
Anishinaabe and Haudenosaunee Territory.<https://www.queensu.ca/encyclopedia/t/traditional-territories>
________________________________

**Comment by carltape** [comment]
Greetings! The next specfem zoom hour is next Wed (Feb 7th) at
0800 Alaska time
1200 US Eastern
1800 central Europe time
The zoom link will be sent to anyone who has signed up via the link at the top of the thread here. If you signed up once before, you don't need to do so again. If you want to add the specfem monthly meetings to your own calendar, please see details above. Come join if you have questions or suggestions about any of the specfem packages. Agenda TBD -- feel free to add items or make requests here. UAF postdoc Evans Onyango will be the host. Reminder about the forthcoming [specfem hybrid workshop](https://github.com/orgs/SPECFEM/discussions/1664) on the afternoon of Tuesday April 30th at SSA2024 in Anchorage, Alaska.

**Comment by carltape** [comment]
Greetings! The next specfem zoom hour is next Wed (Mar 6th) at
0800 Alaska time
1200 US Eastern
1800 central Europe time
The zoom link will be sent to anyone who has signed up via the link at the top of the thread here. If you signed up once before, you don't need to do so again. If you want to add the specfem monthly meetings to your own calendar, please see details above. Come join if you have questions or suggestions about any of the specfem packages. Agenda TBD -- feel free to add items or make requests here. Xiongwei Liu from U. Colorado will share some intriguing slides about kernel features in the vicinity of sources and receivers.

Reminder about the forthcoming [specfem hybrid workshop](https://github.com/orgs/SPECFEM/discussions/1664) on the afternoon of Tuesday April 30th at SSA2024 in Anchorage, Alaska.

**Comment by bch0w** [comment]
Hi SPECFEM community, the next SPECFEM Zoom hour will be this Wednesday, April 3rd, and I would like to spend some of our meeting discussing the upcoming SPECFEM Developers Workshop (#1664), taking place both in-person and on Zoom April 30th at SSA2024. 

Please attend the Zoom this Wednesday and join in on discussing a potential agenda, where I hope to get some input on:
1. How can we make this a useful workshop for all?
2. What is the best use of our limited time, e.g., lightning talks, open discussion, mini-hackathon?
3. Are there any goals we want to accomplish? 

Reminder that the workshop is hybrid and registration is open, please sign up if you haven't: https://geodynamics.org/events/details/343 

Hope to see you all this Wednesday for the monthly zoom! And at the workshop April 30th!

**Comment by carltape** [comment]
Greetings! The next specfem zoom hour is next Wed (May 8th) at
0800 Alaska time
1200 US Eastern
1800 central Europe time
The zoom link will be sent to anyone who has signed up via the link at the top of the thread here. If you signed up once before, you don't need to do so again. If you want to add the specfem monthly meetings to your own calendar, please see details above. Come join if you have questions or suggestions about any of the specfem packages. Agenda TBD -- feel free to add items or make requests here.

It was great to see many of you in person and online at the specfem hybrid workshop in Anchorage (SSA 2024). Many thanks to Bryant, Qinya, and Lorraine for hosting this. And thanks to all the presenters.

**Comment by carltape** [comment]
Greetings! The next specfem zoom hour is tomorrow/Wed (June 5th) at
0800 Alaska time
1200 US Eastern time
1800 central Europe time
The zoom link will be sent to anyone who has signed up via the link at the top of the thread here. If you signed up once before, you don't need to do so again. If you want to add the specfem monthly meetings to your own calendar, please see details above. Come join if you have questions or suggestions about any of the specfem packages. Agenda TBD -- feel free to add items or make requests here.

**Reply by tianshi-liu** [reply]
I'm hoping to discuss our implementation of wavefield injection, with more details than my SPECFEM workshop presentation. Currently the code can be accessed at https://github.com/tianshi-liu/specfem3D-injection.

**Comment by carltape** [comment]
Greetings! The next specfem zoom hour is next Wed (July 3rd) at
0800 Alaska time
1200 US Eastern
1800 central Europe time
The zoom link will be sent to anyone who has signed up via the link at the top of the thread here. If you signed up once before, you don't need to do so again. If you want to add the specfem monthly meetings to your own calendar, please see details above. Come join if you have questions or suggestions about any of the specfem packages. Agenda TBD -- feel free to add items or make requests here.

**Reply by Rohit-Kakodkar** [reply]
I wanted to give a short update on SPECFEM++ and the new adjoint capabilities in the package.

**Reply by tianshi-liu** [reply]
I'm hoping to very briefly discuss a diffusion-equation-based smoothing program that I'm going to contribute to SPECFEM3D_Cartesian if time permits. ~5mins should be enough.

**Comment by carltape** [comment]
Greetings! The next specfem zoom hour is next Wed (Aug 7th) at
0800 Alaska time
1200 US Eastern
1800 central Europe time
The zoom link will be sent to anyone who has signed up via the link at the top of the thread here. If you signed up once before, you don't need to do so again. If you want to add the specfem monthly meetings to your own calendar, please see details above. Come join if you have questions or suggestions about any of the specfem packages. Agenda TBD -- feel free to add items or make requests here.

**Reply by bch0w** [reply]
Two items for the agenda from me:

1. SPECFEM movie visualization source region high-frequency artefacts
2. OMPI-Cuda seg fault error when running multiple multi-gpu forward simulations

**Comment by carltape** [comment]
Greetings! The next specfem zoom hour is this Wed (Sept 4th) at
0800 Alaska time
1200 US Eastern
1800 central Europe time
The zoom link will be sent to anyone who has signed up via the link at the top of the thread here. If you signed up once before, you don't need to do so again. If you want to add the specfem monthly meetings to your own calendar, please see details above. Come join if you have questions or suggestions about any of the specfem packages. Agenda TBD -- feel free to add items or make requests here.

**Comment by carltape** [comment]
Greetings! The next specfem zoom hour is tomorrow/Wed (Oct 2nd) at
0800 Alaska time
1200 US Eastern
1800 central Europe time
The zoom link will be sent to anyone who has signed up via the link at the top of the thread here. If you signed up once before, you don't need to do so again. If you want to add the specfem monthly meetings to your own calendar, please see details above. Come join if you have questions or suggestions about any of the specfem packages. Agenda TBD -- feel free to add items or make requests here.

**Reply by tianshi-liu** [reply]
I'm hoping to ask a question about SPECFEM simulations with strong attenuation if time permitted

**Comment by carltape** [comment]
Greetings! The next specfem zoom hour is Wed (Nov 6th) at
0800 Alaska time
1200 US Eastern
1800 central Europe time
The zoom link will be sent to anyone who has signed up via the link at the top of the thread here. If you signed up once before, you don't need to do so again. If you want to add the specfem monthly meetings to your own calendar, please see details above. Come join if you have questions or suggestions about any of the specfem packages. Feel free to add items or make agenda requests in this github discussion.

This week Rohit will be presenting on the new release of SPECFEM++.  This release adds adjoint capabilities for 2D domains within SPECFEM++ and adds performance optimizations across multiple architecture. I encourage you to try the code.

Documentation: [https://specfem2d-kokkos.readthedocs.io/en/latest/index.html#](https://specfem2d-kokkos.readthedocs.io/en/latest/index.html)
Code: [https://github.com/PrincetonUniversity/SPECFEMPP](https://github.com/PrincetonUniversity/SPECFEMPP?tab=readme-ov-file)

**Reply by bch0w** [reply]
Some additional agenda Items:
- CIG TNG meeting, August 2025
- SPECFEM/SPECFEM++ tutorials at CIG TNG?

**Comment by carltape** [comment]
Greetings! The next specfem zoom hour is Wed (Dec 4th) at
0800 Alaska time
1200 US Eastern
1800 central Europe time
The zoom link will be sent to anyone who has signed up via the link at the top of the thread here. If you signed up once before, you don't need to do so again. If you want to add the specfem monthly meetings to your own calendar, please see details above. Come join if you have questions or suggestions about any of the specfem packages. Feel free to add items or make agenda requests in this github discussion.

**Comment by carltape** [comment]
Greetings! The next specfem zoom hour is Wed (Jan 8th) at
0800 Alaska time
1200 US Eastern
1800 central Europe time
The zoom link will be sent to anyone who has signed up via the link at the top of the thread here. If you signed up once before, you don't need to do so again. If you want to add the specfem monthly meetings to your own calendar, please see details above. Come join if you have questions or suggestions about any of the specfem packages. Feel free to add items or make agenda requests in this github discussion.

**Comment by carltape** [comment]
Greetings! The next specfem zoom hour is Wed (Feb 5th) at
0800 Alaska time
1200 US Eastern
1800 central Europe time
The zoom link will be sent to anyone who has signed up via the link at the top of the thread here. If you signed up once before, you don't need to do so again. If you want to add the specfem monthly meetings to your own calendar, please see details above. Come join if you have questions or suggestions about any of the specfem packages. Feel free to add items or make agenda requests in this github discussion.

Bryant Chow and I will not be able to join on account of a teaching conflict. Thanks to Aakash Gupta for agreeing to lead the call.

**Comment by carltape** [comment]
Hi, everyone. Both Bryant and I are unavailable on Wed, as we'll be doing a GPR survey of lake ice for a geophysics class. Is anyone willing and interested to lead the call this month? If so, I can send out the usual reminders, and the same zoom link will work.

**Reply by aakash10gupta** [reply]
I can lead the meeting.

**Comment by carltape** [comment]
Greetings! The next specfem zoom hour is Wed (April 2nd) at
0800 Alaska time
1200 US Eastern
1800 central Europe time
The zoom link will be sent to anyone who has signed up via the link at the top of the thread here. If you signed up once before, you don't need to do so again. If you want to add the specfem monthly meetings to your own calendar, please see details above. Come join if you have questions or suggestions about any of the specfem packages. Feel free to add items or make agenda requests in this github discussion.

**Comment by carltape** [comment]
Greetings! The next specfem zoom hour is Wed/tomorrow (May 7th) at
0800 Alaska time
1200 US Eastern
1800 central Europe time
The zoom link will be sent to anyone who has signed up via the link at the top of the thread here. If you signed up once before, you don't need to do so again. If you want to add the specfem monthly meetings to your own calendar, please see details above. Come join if you have questions or suggestions about any of the specfem packages. Feel free to add items or make agenda requests in this github discussion. And please share your research progress and any recent specfem publications!

**Comment by carltape** [comment]
Greetings! The next specfem zoom hour is Wed (June 4th) at
0800 Alaska time
1200 US Eastern
1800 central Europe time
The zoom link will be sent to anyone who has signed up via the link at the top of the thread here. If you signed up once before, you don't need to do so again. If you want to add the specfem monthly meetings to your own calendar, please see details above. Come join if you have questions or suggestions about any of the specfem packages. Feel free to add items or make agenda requests in this github discussion. And please share your research progress and any recent specfem publications!

**Comment by carltape** [comment]
Greetings! The next specfem zoom hour is Wed (July 2nd) at
0800 Alaska time
1200 US Eastern
1800 central Europe time
The zoom link will be sent to anyone who has signed up via the link at the top of the thread here. If you signed up once before, you don't need to do so again. If you want to add the specfem monthly meetings to your own calendar, please see details above. Come join if you have questions or suggestions about any of the specfem packages. Feel free to add items or make agenda requests in this github discussion. And please share your research progress and any recent specfem publications!

**Reply by Rohit-Kakodkar** [reply]
Hi, I'd like to add SPECFEM++ v0.4.0 release to the agenda today.

**Comment by carltape** [comment]
Greetings! The next specfem zoom hour is Wed (Aug 6th) at
0800 Alaska time
1200 US Eastern
1800 central Europe time
The zoom link will be sent to anyone who has signed up via the link at the top of the thread here. If you signed up once before, you don't need to do so again. If you want to add the specfem monthly meetings to your own calendar, please see details above. Come join if you have questions or suggestions about any of the specfem packages. Feel free to add items or make agenda requests in this github discussion. And please share your research progress and any recent specfem publications!

I realize that our meeting is right in the middle of the [CIG Next Generation meeting in Colorado](https://geodynamics.org/events/details/349/agenda). Good luck to the participants who are presenting on specfem there.

**Comment by carltape** [comment]
Greetings! The next specfem zoom hour is next Wed (Sept 3rd) at
0800 Alaska time
1200 US Eastern
1800 central Europe time
The zoom link will be sent to anyone who has signed up via the link at the top of the thread here. If you signed up once before, you don't need to do so again. If you want to add the specfem monthly meetings to your own calendar, please see details above. Come join if you have questions or suggestions about any of the specfem packages. And please share your research progress and any recent specfem publications!

Tentative agenda items:
+ importing radial anisotropy models from EMC into specfem_globe
+ I/O updates for massive source files (Qinya, Nanqiao)

Please add to this discussion with items for the agenda. Or just come and be ready to share. Thanks.

**Reply by carltape** [reply]
Maybe a recap from those of you who attended the CIG meeting in Colorado last month?

**Comment by carltape** [comment]
Greetings! The next specfem zoom hour is **Wed/tomorrow** (Oct 1st) at
0800 Alaska time
1200 US Eastern
1800 central Europe time
The zoom link will be sent to anyone who has signed up via the link at the top of the thread here. If you signed up once before, you don't need to do so again. If you want to add the specfem monthly meetings to your own calendar, please see details above. Come join if you have questions or suggestions about any of the specfem packages. And please share your research progress and any recent specfem publications!

**Comment by carltape** [comment]
Greetings! The next specfem zoom hour is **Wed (Nov 5th)** at
0800 Alaska time
1200 US Eastern
1800 central Europe time
The zoom link will be sent to anyone who has signed up via the link at the top of the thread here. If you signed up once before, you don't need to do so again. If you want to add the specfem monthly meetings to your own calendar, please see details above. Come join if you have questions or suggestions about any of the specfem packages. And please share your research progress and any recent specfem publications!

**Comment by carltape** [comment]
Greetings! The next specfem zoom hour is Wed (Dec 3rd) at
0800 Alaska time
1200 US Eastern
1800 central Europe time
The zoom link will be sent to anyone who has signed up via the link at the top of the thread here. If you signed up once before, you don't need to do so again. If you want to add the specfem monthly meetings to your own calendar, please see details above. Come join if you have questions or suggestions about any of the specfem packages. And please share your research progress and any recent specfem publications!

Also, people should share their AGU plans on the call. Jeroen Tromp is giving the 2025 Gutenberg Lecture, which is definitely going to contain some specfem highlights. And Heiner Igel is being honored as an AGU Fellow.

**Comment by carltape** [comment]
Greetings! The next specfem zoom hour is Wed (Jan 14th) at
0800 Alaska time
1200 US Eastern
1800 central Europe time
The zoom link will be sent to anyone who has signed up via the link at the top of the thread here. If you signed up once before, you don't need to do so again. If you want to add the specfem monthly meetings to your own calendar, please see details above. Come join if you have questions or suggestions about any of the specfem packages. And please share your research progress and any recent specfem publications!

**Comment by ljhwang** [comment]
Hi

AxiSEM will be holding its first community development workshop MonTue April 13-14 in Pasadana, CA - before SSA. If you are interested in being part of this community and contributing to AxiSEM development, please consider joining. For more details, see:

	https://geodynamics.org/events/details/384

We will also hope to have a SIG at SSA.

And apologies for spamming the list.

Best,
-Lorraine

*****************************
Lorraine Hwang, Ph.D. - Co Director
Computational Infrastructure for Geodynamics
UC Davis
530.752.3656
geodynamics.org

**Comment by carltape** [comment]
Greetings! The next specfem zoom hour is Wed (Feb 4th) at
0800 Alaska time
1200 US Eastern
1800 central Europe time
The zoom link will be sent to anyone who has signed up via the link at the top of the thread here. If you signed up once before, you don't need to do so again. If you want to add the specfem monthly meetings to your own calendar, please see details above. Come join if you have questions or suggestions about any of the specfem packages. And please share your research progress and any recent specfem publications!

Rohit and Co. will first present the release notes and then hand it over to Kentaro Hanson to present his work on incorporating the discontinuous Galerkin method into SPECFEM++ for non-conforming interfaces.

**Comment by carltape** [comment]
Greetings! The next specfem zoom hour is Wed/today (Mar 4th) at
0800 Alaska time
1200 US Eastern
1800 central Europe time
The zoom link will be sent to anyone who has signed up via the link at the top of the thread here. If you signed up once before, you don't need to do so again. If you want to add the specfem monthly meetings to your own calendar, please see details above. Come join if you have questions or suggestions about any of the specfem packages. And please share your research progress and any recent specfem publications!

**Comment by carltape** [comment]
Greetings! The next specfem zoom hour is Wed (April 1st) at
0800 Alaska time
1200 US Eastern
1800 central Europe time
The zoom link will be sent to anyone who has signed up via the link at the top of the thread here. If you signed up once before, you don't need to do so again. If you want to add the specfem monthly meetings to your own calendar, please see details above. Come join if you have questions or suggestions about any of the specfem packages. And please share your research progress and any recent specfem publications!

**Comment by carltape** [comment]
Greetings! The next specfem zoom hour is Wed (May 6th) at
0800 Alaska time
1200 US Eastern
1800 central Europe time
The zoom link will be sent to anyone who has signed up via the link at the top of the thread here. If you signed up once before, you don't need to do so again. If you want to add the specfem monthly meetings to your own calendar, please see details above. Come join if you have questions or suggestions about any of the specfem packages. And please share your research progress and any recent specfem publications!

**Comment by zohreh2206** [comment]
Hi

I signed up for todays meeting but didn't get log in link yet.

Bests,

Zohreh Abbasi

she/her/hers

[A logo with blue letters and a black background  Description automatically generated]

University of Memphis Student

PhD candidate | Geophysics
Class of 2026
Center for Earthquake Research and Information

[UofM Facebook]<https://www.facebook.com/uofmemphis>  [UofM Twitter] <https://twitter.com/uofmemphis>   [UofM Instagram] <https://www.instagram.com/uofmemphis/>



________________________________

**Comment by bch0w** [comment]
Greetings! The next specfem zoom hour is Wed (June 3rd) at
0800 Alaska time
1200 US Eastern
1800 central Europe time
The zoom link will be sent to anyone who has signed up via the link at the top of the thread here. If you signed up once before, you don't need to do so again. If you want to add the specfem monthly meetings to your own calendar, please see details above. Come join if you have questions or suggestions about any of the specfem packages. And please share your research progress and any recent specfem publications!

**Comment by carltape** [comment]
Greetings! The next specfem zoom hour is Wed/tomorrow (July 1st) at
0800 Alaska time
1200 US Eastern
1800 central Europe time
The zoom link will be sent to anyone who has signed up via the link at the top of the thread here. If you signed up once before, you don't need to do so again. If you want to add the specfem monthly meetings to your own calendar, please see details above. Come join if you have questions or suggestions about any of the specfem packages. And please share your research progress and any recent specfem publications!

**Comment by carltape** [comment]
Greetings! The next specfem zoom hour is Wed/tomorrow (Aug 5th) at
0800 Alaska time
1200 US Eastern
1800 central Europe time
The zoom link will be sent to anyone who has signed up via the link at the top of the thread here. If you signed up once before, you don't need to do so again. If you want to add the specfem monthly meetings to your own calendar, please see details above. Come join if you have questions or suggestions about any of the specfem packages. And please share your research progress and any recent specfem publications!

**Comment by carltape** [comment]
Greetings! The next specfem zoom hour is Wed/tomorrow (Sept 2nd) at
0800 Alaska time
1200 US Eastern
1800 central Europe time
The zoom link will be sent to anyone who has signed up via the link at the top of the thread here. If you signed up once before, you don't need to do so again. If you want to add the specfem monthly meetings to your own calendar, please see details above. Come join if you have questions or suggestions about any of the specfem packages. And please share your research progress and any recent specfem publications!

**Reply by lsawade** [reply]
We'll give an update on the latest SPECFEM++ release: [SPECFEM++ v0.6.0](https://github.com/PrincetonUniversity/SPECFEMPP/releases/tag/v0.6.0)
